### DISCUSSION TITLE
Reduce the reputation requirement for WSG/AB epic items to revered/honored as a form of a catch up mechanic.

### DISCUSSION THREAD

**Discussion by SadFatBear** [chromiecraft/chromiecraft#3352] [Suggestions]
With the lauch of BWL the PvP epic items lost a lot of their initial appeal and the requirement of exalted reputation does not incentivise players to even atempt to grind for them. Therefore I suggest to reduce the reputation required to revered (or maybe even honored, like AV) so that players can put those items to at least some use before they become completely obsolete.

If you decide to reduce the requirements now you could do it again once AQ launches as a form of catch up mechanic and then drop it completely with TBC launch to make it 100% wotlk-like.

**Comment by Annamaria-CC** [comment]
Hey SadFatBear,

Every time we release a new bracket, we look at the older introduced mechanics. We will be looking into pvp rep requirements and see if they need adjustments during Zul Gurub launch.

**Reply by Vitusin** [reply]
Hope that means ZG is ,,soon".

**Comment by Annamaria-CC** [comment]
This suggestion has been accepted.
